### DISCUSSION TITLE
whats difference between two controlnet one by lllyasviel and another by Mikubill

### DISCUSSION THREAD

**Discussion by AmitShashi** [Mikubill/sd-webui-controlnet#973] [Q&A] [answered]
whats difference between two controlnet one by lllyasviel and another by Mikubill
which one i should use
which one is latest
do installing both of them at same time can cause any issue.
thanks in advance for replying.

**Comment by marcsyp** [comment, marked as answer]
This repo by mikubill is the Automatic1111 extension for the standalone ControlNet project (by illyasviel), it is essentially a wrapper for ControlNet so it can be used in the GUI.  You only need to install this project if you intend to use CN in the browser extension.

**Reply by remo-thanks** [reply]
I am still having trouble comprehending.
